zng/docs/spec.md

```diff
@@ -110,6 +110,16 @@ Note that the value encoding need not refer to the field names and types as that
 completely captured by the type ID.  Values merely encode the value
 information consistent with the referenced type ID.
 
+Since type definitions are carried in-band in a ZNG stream,
+the entire stream must generally be processed in-order to maintain
+accurate and complete information on types.
+To facilitate random access into large stored ZNG streams, a stream
```

Comment:
s/To facilitate random access/to enable bi-directional reading and facilitate random access/?